packages/calypso-products/src/constants/wpcom.ts

```diff
@@ -26,6 +26,7 @@ export const PLAN_BLOGGER_2_YEARS = 'blogger-bundle-2y';
 export const PLAN_ECOMMERCE_MONTHLY = 'ecommerce-bundle-monthly';
 export const PLAN_ECOMMERCE = 'ecommerce-bundle';
 export const PLAN_ECOMMERCE_2_YEARS = 'ecommerce-bundle-2y';
+export const PLAN_ECOMMERCE_TRIAL_MONTHLY = 'wp-bundle-ecommerce-trial';
```

Comment:
As per my post for the creation of the new product (paYKcK-2zK-p2), this is the product slug and should be `ecommerce-trial-bundle-monthly`.
```suggestion
export const PLAN_ECOMMERCE_TRIAL_MONTHLY = 'ecommerce-trial-bundle-monthly';
```